recipes/pykml/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
You do not need this file since you have the build: script: entry in the .yaml file.